gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-plans.spec.ts

```diff
@@ -0,0 +1,187 @@
+import { ADMIN_USER, API_PUBLISHER_USER } from '@fakers/users/users';
+import { ApisFaker } from '@gravitee/fixtures/management/ApisFaker';
+import { PlansFaker } from '@gravitee/fixtures/management/PlansFaker';
+import { ApiEntity, PlanStatus } from '../../../../../lib/management-webclient-sdk/src/lib/models';
+import faker from '@faker-js/faker';
+
+import homePage from "@pageobjects/HomePage/HomePage"
+import apiHome from "@pageobjects/Apis/ApiHome"
+import apiDetails from '@pageobjects/Apis/ApiDetails';
+
+// This is purely for PageObject example and on hold for now until policy studio changes are completed.
+
+
+    describe('API Plans Feature', () => {
+        beforeEach(() => {
+          cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+        });
+
+        const HomePage = new homePage()
+        const ApiHome = new apiHome()
+        const ApiDetails = new apiDetails()
+
+        it('Verify Home Page', () => {
+            cy.visit('/')
+            cy.wait(1000)
+            cy.contains('Home board').should('be.visible');
+        });
+        
+        it('Verify Existing Plan', () => {
+            cy.visit('/#!/environments/default/apis/?q=Test&page=1&size=10');
+            cy.getByDataTestId('search').click().type("Test");
+            cy.get('input').type('{enter}')
+            cy.wait(1000)
+            cy.getByDataTestId('api_list_edit_button').first().click();
+            cy.contains('Design').should('be.visible');
+            cy.contains('General').should('be.visible');
+            cy.contains('Plans').should('be.visible');
+            ApiDetails.plans().click();
+            cy.contains('STAGING').should('be.visible');
+            cy.contains('PUBLISHED').should('be.visible');
+            cy.contains('DEPRECATED').should('be.visible');
+            cy.contains('CLOSED').should('be.visible');
+            cy.contains('Default Keyless').should('be.visible')
+            // I don't know if this is liable to change ... I am assuming we set up and look for / use same API every time.
+        })
+
+        it('Create a generic New Plan (API Key), verify and delete', () => {
+            cy.visit('/#!/environments/default/apis/?q=Test&page=1&size=10');
+            cy.getByDataTestId('api_list_edit_button').first().click();
+            ApiDetails.plans().click();
+            cy.contains('STAGING').should('be.visible');
+            cy.contains('PUBLISHED').should('be.visible');
+            cy.contains('DEPRECATED').should('be.visible');
+            cy.contains('CLOSED').should('be.visible');
+
+            cy.getByDataTestId('api_addPlan_button').click();
+            cy.contains('API Key').click();
+
+            cy.getByDataTestId('api_planName').type('TestApiKey')
+            cy.getByDataTestId('api_planDescription').type('Testy Test McTest')
+            // ^ ^ Could probably do with being faker'd
```

Comment:
Yes, I agree Faker would be a good option.

Reply:
Need to push working changes, then pull master and work on faker separately because of faker update